t/t2900-job-timelimits.t

```diff
@@ -19,6 +19,12 @@ test_expect_success 'job time limits are enforced' '
 		flux run --time-limit=${TIMEOUT}s sleep 30 2>limit1.err &&
 	grep "resource allocation expired" limit1.err
 '
+test_expect_success 'tiemout exception was generated by instance owner' '
```

Comment:
s/tiemout/timeout/

Reply:
Doh! I had fixed that but I guess I forgot to push it.   Pushed it now and I'll set MWP.  Thanks!